packages/webapp/src/containers/Animals/Inventory/index.tsx

```diff
@@ -231,6 +240,12 @@ function AnimalInventory({ isCompactSideMenu }: AnimalInventoryProps) {
               }}
             />
           ) : null}
+          <RemoveAnimalsModal
+            isOpen={removalModalOpen}
+            onClose={() => setRemovalModalOpen(false)}
+            onConfirm={handleAnimalOrBatchRemoval}
+            showSuccessMessage={false}
```

Comment:
Success message mentions historical inventory, so I am assuming it should not be shown before that has been built out.